WalletWasabi.Fluent.Desktop/WalletWasabi.Fluent.Desktop.csproj

```diff
@@ -24,8 +24,8 @@
 	</PropertyGroup>
 
 	<PropertyGroup>
-		<Authors>nopara73;lontivero;danwalmsley</Authors>
-		<Company>zkSNACKs</Company>
+		<Authors>zkSNACKs Ltd</Authors>
```

Comment:
```suggestion
		<Authors>https://github.com/zkSNACKs/WalletWasabi/graphs/contributors</Authors>
```

Reply:
I am not sure if we should change this at all. I think we can keep the original or update to WW2 with the actual Authors (link above).